plot_test.go

```diff
@@ -56,10 +56,10 @@ func TestLegendAlignment(t *testing.T) {
 		},
 		&recorder.Fill{
 			Path: vg.Path{
-				{Type: vg.MoveComp, X: 80, Y: 30},
-				{Type: vg.LineComp, X: 80, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 30},
+				{Type: vg.MoveComp, X: 80, Y: 31.865107548701296},
```

Comment:
Instead of this, let's change the magic constant on line 22 until the Y values match the expected. I'm not sure who (I suspect @kortschak) came up with this number or how, but I wonder if using `10` would work. If so, I'd feel quite confident in this change.